test/vaadin-radio-button.html

```diff
@@ -44,7 +44,7 @@
 
   <test-fixture id="default">
     <template>
-      <vaadin-radio-button name="test-radio">Radio <b>Button</b></vaadin-radio-button>
+      <vaadin-radio-button name="test-radio">Radio <span>Button</span></vaadin-radio-button>
```

Comment:
The tests and the fix seem to do the job. However, I wasn't able to reproduce the original issue using  a `<span>`. To properly reproduce, I needed to use `<label>` as was done in the [issue description](https://github.com/vaadin/web-components/issues/7128#issue-2132149796).

https://github.com/vaadin/vaadin-radio-button/assets/1222264/10324583-4500-4aaa-9da1-ef9a83a1ac6f

Reply:
Ah, yeah, this is something that I intended to mention but forgot. I've considered that using label is technically not correct given that the radio button already has its own label inside the shadow DOM.

Reply:
No need. As long as the tests don't report false positives, a `<span>` is fine.